### DISCUSSION TITLE
How can I write the attribute value of a cluster without using Dev console?

### DISCUSSION THREAD

**Discussion by amigthea** [Koenkk/zigbee2mqtt#20508] [General]
I need to do that from the terminal with the `mosquitto_pub` command but I do not know the syntax

**Comment by Koenkk** [comment]
Publish to `zigbee2mqtt/DEVICE_NAME/set` payload `{"cluster":"CLUSTER", "payload": [{"ATTRIBUTE_NAME": "VALUE"}]}`

**Reply by amigthea** [reply]
thank you <3
do you have a link to the mqtt docs where I can learn those type of syntax? have a nice day anyway!
